packages/runtime-core/src/components/BaseTransition.ts

```diff
@@ -439,7 +442,9 @@ export function resolveTransitionHooks(
     },
 
     clone(vnode) {
-      return resolveTransitionHooks(vnode, props, state, instance)
+      const hooks = resolveTransitionHooks(vnode, props, state, instance)
```

Comment:
should `postClone` be passed here as well?

Reply:
It shouldn't be necessary. Each time the inner child of a Transition are switched, a new `enterHooks` is created. I can't think of any scenario where the cloned hooks would be cloned again. 🤔